### PR TITLE
Get Sequences By Criteria

### PR DESCRIPTION
**Pull request by vivbak**

### CODE REVIEW

**Comment by vivbak** [comment]
Oops sorry @illusional I thought I requested a review on this already 😅

File: api/routes/sequence.py [hunk 1]
```diff
@@ -128,6 +130,46 @@ async def get_sequence(
...
 
 
+@router.post('/criteria', operation_id='getSequencesByCriteria')
+async def get_sequences_by_criteria(
+    sample_ids: List[str] = None,
+    sequence_ids: List[int] = None,
+    seq_meta: Dict = None,
+    sample_meta: Dict = None,
+    project_ids: List[str] = None,
+    active: bool = Body(default=True),
```
Comment (illusional) [inline api/routes/sequence.py:140]: Why Body here, seems reasonable to keep this in the query params.

**Review by illusional — CHANGES_REQUESTED** [review]

**Comment by illusional** [comment]
Looks really great @vivbak, I've left a few questions + just need to make sure if project_ids aren't specified, we're correctly checking permissions on returned resources.

**Review by illusional — APPROVED** [review]
Amazing work! Can you add a bumpversion commit, and then this should be all sweet :)

**Pull request merged**